app/models/mixins/relationship_mixin.rb

```diff
@@ -54,6 +55,22 @@ def clear_relationships_cache(*args)
     @association_cache.delete(:all_relationships)
   end
 
+  # this allows us to call similarly named ancestry or mixin methods
+  # depending on whether default_relationship_type is set to :genealogy or not
+
+  def call_ancestry_method(method_name, *args)
```

Comment:
```suggestion
  def call_ancestry_method(method_name, args)
```

Reply:
Sorry, but I don't believe you about this one. 
`ArgumentError: wrong number of arguments (given 1, expected 2)`